sktime/transformations/series/scaledlogit.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implmenents the scaled logit tranformation."""
+
+__author__ = ["ltsaprounis"]
+__all__ = ["ScaledLogitTransformer"]
+
+from copy import deepcopy
+
+import numpy as np
+
+from sktime.transformations.base import BaseTransformer
+
+
+class ScaledLogitTransformer(BaseTransformer):
+    r"""Scaled logit transform.
+
+    Combined with a sktime.forecasting.compose.TransformedTargetForecaster it ensures
```

Comment:
this should primarily describe the transformer. What is this?

That it is very useful in forecasting can be a second paragraph.

Reply:
description should not refer to the "forecast", but to the inputs of the transform. The second paragraph can be specific to its use in forecasting.